macos/FluentUI/Button/Button.swift

```diff
@@ -352,6 +358,14 @@ open class Button: NSButton {
 	}
 
 	private func setColorValues(forStyle: ButtonStyle, accentColor: NSColor) {
+		if increaseContrastEnabled {
+			// Use "UI Element Colors" System Colors which respond correctly to accessibility
+			// settings like increase contrast
+			borderColorRest = .textColor
+			borderColorPressed = .textColor
+			borderColorDisabled = .textColor
```

Comment:
I'm not sure we want to use textColor for borderColor, but I guess you didn't find a better one?

Reply:
Is there a particular reason we wouldn't want to use the textColor for the border? If anything, it should guarantee a strong contrast between the background and the border.

Reply:
Mostly because this is a semantic color, who's name is "textColor" lol. But I can't think of a better color tbh 😅. 

What does Fluent 2 use? Are the text and border color the same on iOS?

Reply:
Maybe controlTextColor vs. textColor?

Reply:
Didn't really find anything else that's border/outline specific.